cpp/oneapi/dal/algo/kmeans/backend/gpu/kernels_fp.hpp

```diff
@@ -20,6 +20,12 @@
 #include "oneapi/dal/backend/primitives/ndarray.hpp"
 #include "oneapi/dal/backend/primitives/distance.hpp"
 
+// #define DEBUG_PRINT
+
+#ifdef DEBUG_PRINT
```

Comment:
Is this a standard definition? I cannot find any other usage of it elsewhere in the repository.

Reply:
Done.